htlcswitch/link.go

```diff
@@ -284,13 +284,16 @@ out:
 		case <-l.channel.UnilateralCloseSignal:
 			log.Warnf("Remote peer has closed ChannelPoint(%v) on-chain",
 				l.channel.ChannelPoint())
-			if err := l.cfg.Peer.WipeChannel(l.channel); err != nil {
-				log.Errorf("unable to wipe channel %v", err)
-			}
 
-			// TODO(roasbeef): need to send HTLC outputs to nursery
+			go func() {
```

Comment:
I don't follow. This commit message doesn't give enough context, detail or background information sufficient for me to understand the rationale behind this change. 

Why is launching this in a goroutine better than launching the call to `Stop` (below) in the diff?